recipe/meta.yaml

```diff
@@ -10,19 +10,18 @@ source:
   sha256: 9d126735b1c1b08a56e02e11d0eba20c040ff934c8d1fd6644e7ee085a420569
 
 build:
-  number: 0
+  number: 1
   # Skipping unnecessary linux platforms that don't need viz.
   skip: True  # [py<36 or ppc64le or s390x]
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
-
   host:
     - python
     - setuptools
     - setuptools_scm
     - wheel
-
+    - pip
   run:
     - python
     - numpy
```

Comment:
pyqt is a run_constrained: https://github.com/glue-viz/glue-vispy-viewers/blob/7ce0c55989eee9dc4056e5ce1547591cbaab86b6/setup.cfg#L39

Reply:
it should be run_constrained, but this is a rebuild, and the inital build had pyqt in run. So I will leave it as is.